src/main/resources/static/views/projects/submissions.html

```diff
@@ -0,0 +1,209 @@
+<body>
+<div class="container-fluid">
+    <div class="row content">
+        <div class="col-sm-9">
+            <div class="row">
+                <div class="col-sm-8">
+                    <h1 class="portalheader">
+                        <i class="icon-cloud-upload" aria-hidden="true"></i> Submissions
+                    </h1>
+                </div>
+                <div class="col-sm-4">
+                    <div class="actionsbar">
+                        <div ng-cloak ng-if="!isProfOrCoordinator">
+                            <button title="AddSubmission" ng-click="addSubmission()" type="button" class="btn btn-success"><i class="icon-doc"></i> Add Submission</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <br>
+            <div class="alert alert-warning" ng-cloak ng-show="!isProfOrCoordinator">
+                <strong>Notification:</strong> The final report is due April 5th at noon.
+            </div>
+            <table id="submissionsTable" class="display" cellspacing="0" width="100%">

```

Comment:
should we be seeing the same table when we log in as student too?

Reply:
Yes it will be the same table. However for profs they will see all project submissions and the action for assigning grades, whereas for students, they get to see the submissions for their project along with the corresponding grade.